server/routes/mentions.js

```diff
@@ -0,0 +1,76 @@
+const router = require("express").Router();
+
+const requiresAuth = require("./middleware/requiresAuth");
+const { Mention, User } = require("../models");
+
+const callScraper = require("../scraper");
+
+// TODO test mentions in action
+
+router.get("/", requiresAuth, async (req, res) => {
+  const user = await User.findByPk(req.user.id);
+  // note that the getter method for n-m can accept options
+  // as if it were a regular query
+  const mentions = await user.getMentions();
+
+  res.json(mentions);
+});
+
+router.get("/:id", requiresAuth, async (req, res) => {
+  const mention = await Mention.findByPk(req.params.id);
+
+  res.json(mention);
+});
+
+router.post("/", requiresAuth, async (req, res) => {
+  // NOTE I think this is better served as just a test route in order to test the scraper
+  // This shouldn't be placed in the final PR as there will be another means to post new data
+  // into the database.
+  const user = await User.findByPk(req.user.id);
+
```

Comment:
I think this route is only used when sending the weekly email, and when a user inputs a company name to get the mentions, those mentions don't need to be stored in DB, otherwise there will be a lot more DB operations. 
Thus, we may want to include the user id in params instead of seaching for the currently logged in user.

Reply:
The weekly email will probably not be handled by a route; I believe it is more likely to be handled by an async background worker, such as the one mentioned in #24.

Reply:
We need to clarify what is the DB for in this app as we discussed on Slack.
But if this route is not for weekly email, but for the display of mentions in the main page, I think we don't need to use the currently logged in user to do the search, instead, we want to use the user input in the search bar.

Reply:
I will remove this code out of the PR in any case. I'm starting to see how some of the things in the PR are extraneous and should be omitted until the time comes up